ogr/services/pagure/release.py

```diff
@@ -35,7 +35,7 @@ def url(self) -> Optional[str]:
 
     @property
     def created_at(self) -> datetime.datetime:
-        return None
+        return datetime.datetime.now
```

Comment:
It looks like you forgot parentheses. This returns a function object, not a datetime. In any case, I'm not sure returning the current time is right here. Probably better to change the type signature to `datetime.datetime | None` here and in the base class to reflect that some implementations don't provide this information.